src/formatDate.js

```diff
@@ -19,7 +25,63 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const arrDate = date.split(fromFormat[fromFormat.length - 1]);
+  let year;
+  let month;
+  let day;
+  const newDate = [];
+
+  for (let i = 0; i < fromFormat.length; i++) {
+    switch (fromFormat[i]) {
+      case 'YY':
+      case 'YYYY':
+        year = arrDate[i];
+        break;
+      case 'MM':
+        month = arrDate[i];
+        break;
+      case 'DD':
+        day = arrDate[i];
+        break;
+      default:
+        break;
+    }
+  }
+
+  for (let i = 0; i < fromFormat.length; i++) {
+    for (let j = 0; j < toFormat.length; j++) {
+      if (fromFormat[i] === 'YYYY' && toFormat[j] === 'YY') {
+        year = year.split('').splice(2).join('');
```

Comment:
You could just use year.slice(2) insted of terrorize that miserable string

Reply:
sorry for terrorizing) 
rewrite this